src/main/java/com/indeed/jiraactions/ActionFactory.java

```diff
@@ -61,13 +71,20 @@ public Action create(final Issue issue) throws IOException {
                 .category(issue.initialValue("category"))
                 .fixversions(issue.initialValue("fixversions"))
                 .dueDate(issue.initialValue("duedate"))
-                .components(issue.initialValue("component"))
+                .components(components.stream()
+                        .map(component -> component.name)
+                        .collect(toList()))
                 .labels(issue.initialValue("labels"))
                 .createdDate(issue.fields.created.toString("yyyy-MM-dd"))
                 .createdDateLong(Long.parseLong(issue.fields.created.toString("yyyyMMdd")))
+                .createdDateTimeLong(Long.parseLong(issue.fields.created.toString("yyyyMMddHHmmss")))
+                .createdDateTimestamp(issue.fields.created.getMillis() / 1000)
                 .lastUpdated(0)
                 .closedDate(0)
-                .resolutionDate(getDateResolved(issue.initialValue("resolutiondate")))
+                .resolutionDate(JiraActionsUtil.parseDateTime(issue.initialValue("resolutiondate")).toString("yyyy-MM-dd"))
```

Comment:
Probably slightly more efficient to calculate `issue.initialvalue("resolutiondate")` once and operate on it multiple times.

Reply:
The way this thing is implemented, it holds all the output-ready versions of fields. So instead of containing a `List<>`, it contains the string-joined version of that list. Something to look into changing in a future enhancement.